### PR TITLE
Feature/add redux detail,create,mypage

### PR DESCRIPTION
**Pull request by cw10108**
### DONE
* add redux to group detail page
* implement backend for adding, deleting user, deleting group
* set will_be_deleted true when clicking delete button
### TODO
* testing frontend and backend
* scheduling for deletion
* image loading

### DONE
* add redux to group create page
* implement backend for create
* implement backend, store for otts
### TODO
* testing frontend and backend
* image loading

##TODO
* add redux to mypage
* set payday to calendar
* test mypage

##Remark
* csrf-exempt, to temporarily avoid 403

### CODE REVIEW

File: backend/subroker/views.py [hunk 1]
```diff
@@ -131,18 +132,17 @@ def group_list(request):
...
-                group_account_bank = req_data['account_bank']
-                group_account_number = req_data['account_number']
-                group_account_name = req_data['account_name']
-                group_leader = request.user
+                group_account_bank = req_data['accountBank']
+                group_account_number = req_data['accountNumber']
+                group_account_name = req_data['accountName']
             #ERR 400 : JSONDecodeErr
             except (JSONDecodeError, KeyError) as e:
                 return HttpResponseBadRequest()
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 maybe we dont need to think this error in this project.

File: backend/subroker/views.py [hunk 2]
```diff
@@ -86,6 +86,7 @@ def logout(request):
         return HttpResponseNotAllowed(['GET'])
 
 #Group
+@csrf_exempt
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 maybe we dont need this?

File: backend/subroker/views.py [hunk 3]
```diff
@@ -168,6 +171,7 @@ def group_list(request):
     else:
         return HttpResponseNotAllowed(['GET', 'POST'])
 
+@csrf_exempt
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 same as above

File: backend/subroker/views.py [hunk 4]
```diff
@@ -237,7 +245,7 @@ def group_detail(request, group_id):
                 "account_number": group.account_number, 
                 "account_name": group.account_name, 
                 "leader": group.leader.id,
-                "current_people": group.current_people
+                "current_people": group.current_people,
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 change name to CamelCase

File: backend/subroker/views.py [hunk 5]
```diff
@@ -250,35 +258,44 @@ def group_detail(request, group_id):
             #ERR 404 : Group Doesn't Exist
             except(Group.DoesNotExist) as e:
                 return HttpResponse(status=404)
-            group.delete()
-            return HttpResponse(status=200)
+            group.will_be_deleted = True
+            group.save()
+            print(f"group: {group}")
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 maybe not need for production

File: backend/subroker/views.py [hunk 6]
```diff
@@ -250,35 +258,44 @@ def group_detail(request, group_id):
...
+            return JsonResponse(response_dict, status=200)
+            # return HttpResponse(status=200)
         #ERR 401 : Not Authenticated
         else:
             return HttpResponse(status=401)
     #ERR 405 : METHOD NOT ALLOWED
     else:
         return HttpResponseNotAllowed(['GET', 'PUT', 'DELETE'])
 
+@csrf_exempt
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 same as above

File: backend/subroker/views.py [hunk 7]
```diff
@@ -308,6 +327,47 @@ def group_add_user(request, group_id):
...
 
+def ott_list(request):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            otts = Ott.objects.all()
+            ott_names = set([ott['ott'] for ott in otts.values()])
+            ott_all_list = [{
+                'name': name,
+            } for name in ott_names]
+            print(ott_all_list)
```
Comment (arclic) [inline backend/subroker/views.py]: @cw10108 maybe not need for production

**Review by arclic — APPROVED** [review]

**Review by muhwagwa — APPROVED** [review]

**Pull request merged**